demo/CMakeLists.txt

```diff
@@ -28,9 +27,13 @@ endif(NOT OPENGL_FOUND)
 #########################################################
 # Include Files
 #########################################################
+set(PEGASUS_INCLUDE_DIRS
+    ../../.
```

Comment:
This should be in some `PegasusConfig.cmake` alongside other `PEGASUS_` variables